syntaxes/Junos.tmLanguage

```diff
@@ -229,7 +229,7 @@
+			<string>(?&lt;=^|\s)(logical-systems|\sdynamic-profiles|\sjsrc-partition|\spartition|\sfilter input|\sfilter output|\saccess-profile|\sdscp|\sdscp-ipv6|\sexp|\sieee-802\.1|\sieee-802\.1ad|\sinet-precedence|\sscheduler-map|\sscheduler-maps|\sinput-traffic-control-profile-remaining|\sinput-traffic-control-profile|\straffic-control-profiles|\soutput-traffic-control-profile-remaining|\soutput-traffic-control-profile|\soutput-forwarding-class-map|\sscheduler-map-chassis|\sfragmentation-maps|\ssource-prefix-list|\sbridge-domains|\sgroup|\smime-pattern|\surl-pattern|\slabel-switched-path|\sadmin-groups|\scustom-url-category|\sprofile|\surl-whitelist|\surl-blacklist|\sca-profile|\sidp-policy|\sactive-policy|\sinterface-set|\sinterface-range|\scount|\sdestination-prefix-list|\sschedulers|\sdrop-profiles|\sforwarding-class|\sforwarding-class-map|\simport|\sexport|\sinstance|\sutm-policy|\sids-option|\snext-hop-group|\srouting-instances|\srule|\srule-set|\spool|\sclass|\sunit|\sport-mirror-instance|\sfrom-zone|\sto-zone|\sapply-groups|\sfile|\shost-name|\sdomain-name|\spath|\sdomain-search|\scommunity delete|\scommunity add|\scommunity set|\scommunity|\strap-group|\spolicy|\spolicy-statement|\simport-policy|\sinstance-export|\sinstance-import|\svrf-import|\svrf-export|\simport|\sexport|\skeep-import|\sinter-area-prefix-import|\sinter-area-prefix-export|\snetwork-summary-export|\snetwork-summary-import|\segress-policy|\sbootstrap-import|\sbootstrap-export|\sfilter|\sprefix-list|\sproposal|\saddress-set|\sscheduler|\srib-groups|\sgroups|\ssecurity-zone|\sterm|\sapplication|\sapplication-set|\svlans|\sgateway|\suser|\spolicer|\slsp|\scondition|\sas-path|\sas-path-group|\sservice-set)(?:\s{\s*)</string>
```

Comment:
I've appended `|\sas-path|\sas-path-group|\sservice-set` at the end of pattern. I'll fix if that's a wrong place to add.